README.md

```diff
@@ -1,45 +1,124 @@
+   │                              │  │  │  │                              │
+│  │   zomg, amazing feature A!   │───────▶│         0.42.0-rc.0          │  │
+   │                              │  │  │  │                              │
+│  └──────────────────────────────┘        └──────────────────────────────┘  │
+                   │                 │  │                  │
+│                  ▼                                       ▼                 │
+   ┌──────────────────────────────┐  │  │  ┌──────────────────────────────┐
+│  │                              │        │                              │  │
+   │    whoops, fix feature A     │──┼──┼─▶│    whoops, fix feature A     │
+│  │                              │        │                              │  │
+   └──────────────────────────────┘  │  │  └──────────────────────────────┘
+│                  │                                       │                 │
+                   │                 │  │                  ▼
+│                  │                       ┌──────────────────────────────┐  │
+                   │                 │  │  │                              │
+│                  ▼                       │         0.42.0-rc.1          │  │
+   ┌──────────────────────────────┐  │  │  │                              │
+│  │                              │        └──────────────────────────────┘  │
+   │  wow, sweet new feature B!   │  │  │                  │
+│  │                              │                        ▼                 │
+   └──────────────────────────────┘  │  │  ┌──────────────────────────────┐
+│                  │                       │                              │  │
+                   │                 │  │  │            0.42.0            │
+│                  │                       │                              │  │
+                   │                 │  │  └──────────────────────────────┘
+│                  │                                       │                 │
+                   ▼                 │  │                  ▼
+│  ┌──────────────────────────────┐        ┌──────────────────────────────┐  │
+   │                              │  │  │  │                              │
+│  │  yikes, fix feature A more   │───────▶│  yikes, fix feature A more   │  │
+   │                              │  │  │  │                              │
+│  └──────────────────────────────┘        └──────────────────────────────┘  │
+                   │                 │  │                  │
+│                  │                                       ▼                 │
+                   │                 │  │  ┌──────────────────────────────┐
+│                  │                       │                              │  │
+                   │                 │  │  │         0.42.1-rc.0          │
+│                  │                       │                              │  │
+                   │                 │  │  └──────────────────────────────┘
```

Comment:
Probably we never discussed this, but AFAIK we don't do an RC phase for patch releases post `.0`.

Most likely my mistake in communicating with you the various steps 😅

Reply:
Aha! I may have just filled in the blanks myself 😬